lib/dialects/mssql/data-types.js

```diff
@@ -135,9 +135,13 @@ module.exports = BaseTypes => {
   inherits(DATE, BaseTypes.DATE);
 
   DATE.prototype.toSql = function toSql() {
-    return 'DATETIMEOFFSET';
+    return 'DATETIME';
   };
 
+  DATE.prototype._stringify = function _stringify(date) {
```

Comment:
???

Reply:
Same applies to this comment as I previously mentioned.